sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/ModelsLifecycleSyncSamples.java

```diff
@@ -0,0 +1,120 @@
+ * Decommission the newly created models (model and component) and check for success.
+ * Delete all created models and components.
+ */
+public class ModelsLifecycleSyncSamples {
+
+    private static DigitalTwinsClient client;
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+
+        SamplesArguments parsedArguments = new SamplesArguments(args);
+
+        client = new DigitalTwinsClientBuilder()
+            .tokenCredential(
+                new ClientSecretCredentialBuilder()
+                    .tenantId(parsedArguments.getTenantId())
+                    .clientId(parsedArguments.getClientId())
+                    .clientSecret(parsedArguments.getClientSecret())
+                    .build()
+            )
+            .endpoint(parsedArguments.getDigitalTwinEndpoint())
+            .httpLogOptions(
+                new HttpLogOptions()
+                    .setLogLevel(parsedArguments.getHttpLogDetailLevel()))
+            .buildClient();
+
+        RunModelLifecycleSample();
+    }
+
+    public static void RunModelLifecycleSample() {
+        // For the purpose of this sample we will create temporary models using random model Ids and then decommission a model.
+        // We have to make sure these model Ids are unique within the DigitalTwin instance.
+        String componentModelId = UniqueIdHelper.getUniqueModelId(SamplesConstants.TemporaryComponentModelPrefix, client);
+        String sampleModelId = UniqueIdHelper.getUniqueModelId(SamplesConstants.TemporaryModelPrefix, client);
+
+        String newComponentModelPayload = SamplesConstants.TemporaryComponentModelPayload
+            .replace(SamplesConstants.ComponentId, componentModelId);
+
+        String newModelPayload = SamplesConstants.TemporaryModelWithComponentPayload
+            .replace(SamplesConstants.ModelId, sampleModelId)
+            .replace(SamplesConstants.ComponentId, componentModelId);
+
+        ConsoleLogger.PrintHeader("Create models");
+
+        try {
+            // Create the model
+            // TODO: azabbasi: Discuss the usability of this API with SDK team and change the return type if possible
+            // Currently we have to iterate through the response for the lazy method invocation to run.
+            client.createModels(new ArrayList<String>(Arrays.asList(newComponentModelPayload, newModelPayload)))
+                .forEach((modelData -> {
+                    System.out.println("Created model " + modelData.getId());
+                }));
+
+            ConsoleLogger.PrintSuccess("Created models " + componentModelId + " and " + sampleModelId);
+        }
+        catch (ErrorResponseException ex){
+            if (ex.getResponse().getStatusCode() == HttpStatus.SC_CONFLICT) {
+                ConsoleLogger.PrintWarning("One or more models already existed");
+            }
+        }
+        catch (Exception ex) {
+            ConsoleLogger.PrintFatal("Failed to create models due to: \n" + ex);
```

Comment:
```"Some string " + ex```  doesn't give the full stack trace. That's fine since this is a sample, but for future reference, most loggers take the exception as a parameter after the string like ```log.error("Something bad happened!", someException)```